### PR TITLE
7 landing page

### LINKED ISSUE #7: Create header/menu for landing page

### PR DESCRIPTION
**Pull request by LemonFace0309**
closed #7 

image
image
image

### CODE REVIEW

File: components/Landing/Banner.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import { Container, Grid, Typography, Button, Hidden } from '@material-ui/core';
+import ArrowDownwardIcon from '@material-ui/icons/ArrowDownward';
+
+const Banner = () => {
```
Comment (jimmyygit) [inline components/Landing/Banner.js:4]: Is this the banner? Maybe a rename like Hero would be better? I just think a banner would be better suited to a text and picture strip component. if not then I think banner is ok
image

File: components/Landing/Header.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    threshold: 0,
+    target: window ? window() : undefined,
+  });
+
+  return React.cloneElement(children, {
+    elevation: trigger ? 4 : 0,
+  });
+}
+
+const Header = (props) => {
```
Comment (jimmyygit) [inline components/Landing/Header.js:24]: Is this wip?
image

File: pages/landing/index.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
+import PropTypes from 'prop-types';
+
+import Header from '../../components/Landing/Header';
+import Banner from '../../components/Landing/Banner';
+import LandingSpaces from '../../components/Landing/LandingSpaces';
+import Feature, { DIRECTIONS } from '../../components/Landing/Feature';
+
+const Landing = ({ data, features }) => {
+  return (
+    <div>
```
Comment (jimmyygit) [inline pages/landing/index.js:10]: I think u can just use a fragment here

File: components/Landing/Header.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  const authButtons = classNames([
+    'normal-case',
+    'px-10',
+    'm-2',
+    'rounded-full',
+    'outline-none',
+  ]);
+
+  return (
+    <>
```
Comment (jimmyygit) [inline components/Landing/Header.js:41]: Dont think u need this fragment, theres nothing else adjacent to `ElevationScroll`

**Review by jimmyygit — APPROVED** [review]
image
I think there is some horizontal scrolling here. (Look at the bottom of my screenshot)
Otherwise lgtm

File: components/Landing/Banner.js (hunk 1 above)
Comment (LemonFace0309) [reply components/Landing/Banner.js:4]: Thanks. I actually forgot that it was named Hero. tyty

**Comment by LemonFace0309** [comment]
> image
> I think there is some horizontal scrolling here. (Look at the bottom of my screenshot)
> Otherwise lgtm

thanks, didn't notice that. For future reference, when using `Grid` container element, set `w-full m-0` as styles to avoid this problem.

File: components/Landing/Header.js (hunk 2 above)
Comment (LemonFace0309) [reply components/Landing/Header.js:24]: yes hehe

**Pull request merged**